modes/mu4e/evil-collection-mu4e.el

```diff
@@ -92,6 +92,14 @@ with older release versions of `mu4e.'"
            #'mu4e--main-view-queue)
          args))
 
+(define-obsolete-function-alias 'mu4e~main-toggle-mail-sending-mode
+  'mu4e--main-toggle-mail-sending-mode "1.7.7"
+  "Toggle sending mail mode, either queued or direct.")
```

Comment:
Unless the behavior of these functions has changed significantly, you shouldn't need to provide the docstring. The new function's docstring will be used.